raster/r.kappa/calc_metrics.c

```diff
@@ -213,3 +242,20 @@ static int longcomp(const void *aa, const void *bb)
 
     return (*a > *b);
 }
+
+/* Implements improved Kahan–Babuska algorithm by Neumaier, A. 1974 */
+void update_sum(double *sum, double *c, double value)
+{
+    double tmp;
+
+    tmp = *sum + value;
+
+    if (fabs(*sum) >= fabs(value))
+        *c = *c + (*sum - tmp) + value;
```

Comment:
Any reason not to use the `+=` operator? Even with the extra `*`, `*c += (*sum...` may communicate the intention clearly to both Python and C programmers (operator precedence should be clear or is at least suggested by the formatting). Wikipedia version uses `+=` in its pseudocode example.

Reply:
No reason. Changed to +=